### ISSUE TITLE
startRegionWord and endRegionWord can't contain capital letters in preferences file

### ISSUE THREAD

**Issue by Timmiej93** [thehogfather/brackets-custom-region-code-folding#14]
Since this plugin has been quiet for a long time, this is mostly for reference for other users who stumble upon this issue.

In your preferences file, make sure that the value for `startRegionWord` and `endRegionWord` are all lowercase, otherwise it won't work.

For example:
```
{
	"custom-region-fold.endRegionWord": "regionEnd",
	"custom-region-fold.startRegionWord": "regionStart"
}
```

does not work with

```
// regionStart
    blabla super duper code() {}
// regionEnd
```

If you change your settings file to this, it does work:
```
{
	"custom-region-fold.endRegionWord": "regionend",
	"custom-region-fold.startRegionWord": "regionstart"
}
```

TL;DR: the value for `startRegionWord` and `endRegionWord` in your settings file has to be lowercase. In the code you're using it in, it doesn't matter however, you can use lower and upper case letters throughout.
